*** old.cpp	2020-08-30 18:51:04.082032248 +0800
--- new1.cpp	2020-08-30 19:24:12.263000000 +0800
***************
*** 1,7 ****
  #include <iostream>
  int main(void)
  {
      using namespace std;
!     cout << "hello linux!" << endl;
      return 0;
  }
\ 文件尾没有换行符
--- 1,9 ----
  #include <iostream>
+ #include <stdlib.h>
  int main(void)
  {
      using namespace std;
!     cout << "hello linux! hello diff!" << endl;
!     cout << "test patch!" << endl;
      return 0;
  }
\ 文件尾没有换行符
